tests/series/test_image.py

```diff
@@ -1,14 +1,52 @@
 from __future__ import annotations
 
 import copy
+import io
 
+import cv2
```

Comment:
also add this dep to `requirements-dev`!

Reply:
Ah good catch!